README.md

```diff
@@ -1,2 +1,61 @@
-# Project
-DAY2 BLOCK WEEK1
+# Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+One page website
+
+
+## Built With
+
+- html 
+- css
+- visual studio.NET
+- github
+
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Prerequisites
+VS code
+linters
+browser
+
+### Setup
+copy the url to your local https://github.com/jaamanu/Project.git
+
+## JAMES ASIBEY MANU
+
+👤 **Author1**
```

Comment:
- Kindly change this to Author as you've worked on this project alone. Denoting it with Author1 simply means there are other authors that have collaborated on this project.